.changeset/wet-cows-compare.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": minor
+---
+
+feat: add `Spinner` and `BaseSpinner` components
```

Comment:
I think we should specify internal components because these go in changelog

```suggestion
feat: add `Spinner` component
- Also adds an internal `BaseSpinner` component
```